x-pack/plugins/ml/public/application/settings/calendars/edit/new_event_modal/new_event_modal.js

```diff
@@ -293,13 +299,18 @@ export class NewEventModal extends Component {
           </EuiModalBody>
 
           <EuiModalFooter>
-            <EuiButtonEmpty onClick={closeModal}>
+            <EuiButtonEmpty onClick={closeModal} data-test-subj={'mlCancelCalendarEvent'}>
```

Comment:
To be consistent with other naming, I'd suggest to make this `mlCalendarCancelEventButton`.